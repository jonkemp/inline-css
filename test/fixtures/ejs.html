<body><link rel="stylesheet" href="ejs.css" />
<%= config.sitePrefix %>verify?grant=<%= grant._id %>&secret=<%= grant.secret %>
<div <%= config.sitePrefix %>></div></body>
        