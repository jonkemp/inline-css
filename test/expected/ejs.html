<html><head></head>
<body style="color: red;">
<%= config.sitePrefix %>verify?grant=<%= grant._id %>&amp;secret=<%= grant.secret %>
<div <%= config.sitePrefix %>></div>
        </body></html>
